lib/countUpperCase.js

```diff
@@ -1,11 +1,10 @@
 /**
- * TODO: Write a function that counts all uppercase letters in a string.
+ * A function that counts all uppercase letters in a string.
  * @param str: string with uppercases
  */
 
 function countUpperCase(str) {
-  // Your code goes here.
-  return;
+  return str.length - str.replace(/[A-Z]/g, "").length;
```

Comment:
I think this is backwards. If you remove all the capital letters then you are left counting the lowercase ones.

Reply:
I subtract count of non-uppercase letters from count of all letters. Isn't that right?

Reply:
haha - i must be lost. I didn't even see the first half of the expression earlier. 👍 

I also have a quick tip. Instead of counting what you don't want and subtracting from the the whole string you could count what you _are_ looking for. Check out `String.prototype.match`.